packages/client-utils/src/vstorage.js

```diff
@@ -167,14 +168,12 @@ export const makeVStorage = ({ fetch }, config) => {
       let blockHeight;
       await null;
       do {
-        // console.debug('READING', { blockHeight });
         let values;
         try {
           ({ blockHeight, values } = await vstorage.readAt(
             path,
             blockHeight === undefined ? undefined : Number(blockHeight) - 1,
           ));
-          // console.debug('readAt returned', { blockHeight });
```

Comment:
Any particular reason to get rid of these debug lines? In practice, since `readFully` is so flaky, I use them (temporarily un-comment them) pretty often to figure out what's going on.

Reply:
I agree that there’s no harm in keeping the diagnostic comments. It is nice to have these around when debugging.